cmd/tier-journal.go

```diff
@@ -41,14 +41,27 @@ type tierJournal struct {
 	file     *os.File // active journal file
 }
 
-type jentry struct {
+// jentryV1 represents the entry for V1 of the journal file.
+type jentryV1 struct {
 	ObjName  string `msg:"obj"`
 	TierName string `msg:"tier"`
 }
 
+// jentry represents the entry for the latest version of the journal file.
+type jentry struct {
```

Comment:
Is this breaking? I don't think bumping the version is needed.

When not a tuple, fields can be added/removed just as JSON.

Reply:
Thank you, didn't know that. I've added a serialization/deserialization test for the new struct member addition to confirm this and removed the tier-journal file version change.